packages/@react-spectrum/s2/src/Table.tsx

```diff
@@ -11,85 +11,1110 @@
+    }
+  },
+  paddingBottom: {
+    default: 12, // table-row-bottom-to-text-medium-spacious
+    density: {
+      spacious: size(15), // table-row-bottom-to-text-medium-spacious
+      compact: 8 // table-row-bottom-to-text-medium-compact
+    }
+  },
+  boxSizing: 'border-box',
+  height: 'full',
+  width: 'full',
+  fontSize: 'control',
+  alignItems: 'center',
+  display: 'flex',
+  '--dividerColor': {
+    type: 'borderColor',
+    value: {
+      default: 'gray-300',
+      forcedColors: 'ButtonBorder'
+    }
+  }
+});
+
+const stickyCell = {
+  backgroundColor: 'gray-25'
+} as const;
+
+const checkboxCellStyle = style({
+  ...commonCellStyles,
+  ...stickyCell,
+  paddingStart: 16,
+  paddingEnd: {
+    default: 8,
+    scale: {
+      large: 14
+    }
+  },
+  alignContent: 'center',
+  height: '[calc(100% - 1px)]',
+  borderBottomWidth: 0
+  // TODO: problem with having the checkbox cell itself use the row background color directly instead
+  // of having a separate white rectangle div base below a div with the row background color set above it as a mask
+  // is that it doesn't come out as the same color as the other cells because the base below the sticky cell will be the blue of the
+  // other cells, not the same white base. If I could convert informative-900/10 (and the rest of the rowBackgroundColors) to an equivalent without any opacity
+  // then this would be possible. Currently waiting request for Spectrum to provide tokens for these equivalent values
+  // backgroundColor: '--rowBackgroundColor'
+});
+
+const cellContent = style({
+  truncate: true,
+  whiteSpace: {
+    default: 'nowrap',
+    overflowMode: {
+      wrap: 'normal'
+    }
+  },
+  textAlign: {
+    align: {
+      start: 'start',
+      center: 'center',
+      end: 'end'
+    }
+  },
+  width: 'full',
+  zIndex: 100,
+  padding: {
+    default: 4,
+    isSticky: 0
+  },
+  margin: {
+    default: -4,
+    isSticky: 0
+  }
+});
+
+const cellBackground = style({
+  position: 'absolute',
+  top: 0,
+  bottom: 0,
+  right: 0,
+  left: 0,
+  backgroundColor: {
+    default: 'transparent',
+    isSticky: '--rowBackgroundColor'
+  }
+});
+
+export interface CellProps extends RACCellProps, Pick<ColumnProps, 'align' | 'showDivider'> {
+  /** @private */
+  isSticky?: boolean,
+  /** The content to render as the cell children. */
+  children?: ReactNode
+}
+
+export function Cell(props: CellProps) {
+  let {children, isSticky, showDivider, align, textValue, ...otherProps} = props;
+  let tableVisualOptions = useContext(InternalTableContext);
+  textValue ||= typeof children === 'string' ? children : undefined;
+
+  return (
+    <RACCell
+      // Also isSticky prop is applied just for the layout, will decide what the RAC api should be later
+      // @ts-ignore
+      isSticky={isSticky}
+      // This is a inline style because it needs to set properties ONLY for the end border (don't want to set a color for the bottom border)
```

Comment:
i assume this isn't possible with the theme, but can it be?

Reply:
yeah, can be potentially added to the theme but I didn't add it since it would mean top/bottom/start/end border colors+widths+style would need to be added but I didn't know how many cases would actually use it

Reply:
I think it's fine to add all of those, but @devongovett may have an opinion as well

Reply:
kk, talked about this in stand up, I'll see about adding something for this in the theme

Reply:
Added via https://github.com/adobe/react-spectrum/pull/6778/commits/ae0e6f1eccd86a1220eaded37cf34479aa7188f3